sui/sources/movescription.move

```diff
@@ -132,7 +140,7 @@ module smartinscription::movescription {
     // ======== Functions =========
     fun init(otw: MOVESCRIPTION, ctx: &mut TxContext) {
         let deploy_record = DeployRecord { id: object::new(ctx), version: VERSION, record: table::new(ctx) };
-        do_deploy(&mut deploy_record, b"MOVE", 100_0000_0000, 1704038400*1000, 60*24*15, 100000000, ctx);
+        do_deploy(&mut deploy_record, PROTOCOL_TICK, 100_0000_0000, PROTOCOL_START_TIME_MS, 60*24*15, 100000000, ctx);
```

Comment:
After package upgrade, init function will not run again.

Reply:
Yes, this only clarifies the code, provides the public with functions, and does not change the value.